content/blog/deploying-simple-jitsi-meet-server.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Deploying a simple Jitsi Meet server with NixOS
+date: 2022-07-28
+extra:
+  author: Simon Bruder
+---
+## Introuduction
+
+During the COVID-19 pandemic,
+video conferencing turned out to be an invaluable tool for online collaboration.
+While many used proprietary tools for this,
+there is one proven free and open source video conferencing tool
+that also gained much traction and stayed popular:
+[Jitsi Meet](https://jitsi.org/jitsi-meet/)
+
+Primarily known for its publicly available service [meet.jit.si](https://meet.jit.si),
+it is also possible to self-host Jitsi Meet on your own server.
+
+### Reasons to self-host
+
+There are a variety of reasons,
+why you may consider hosting Jitsi Meet yourself
+instead of relying on an already existing hosted instance:
+
+ * You stay in control of your own data.
+   When relying on a third-party service,
+   you have to trust them
+   and in case you are using the service professionally,
+   you may be legally required to sign a contract on data processing with the provider.
+ * You can have your own branding.
+   Not only does the service run on your own domain,
+   but you can also customise the logo, welcome page etc.
+ * You can configure it to your liking.
+   Depending on the typical use case of the instance,
+   some optional features might be desirable,
+   that are not commonly found on other instances.
+ * You can update the service when you want.
+   If you need the service to work,
+   the user interface totally changing during an important meeting may be problematic.
+   By hosting it yourself, you can decide when to update it.
+
+### Why NixOS
+
+So why deploy this on NixOS and not on Debian or with Docker?
+NixOS has many advantages for running servers.
+It allows you to configure your whole server declaratively,
+so you can re-use the same configuration for multiple servers.
+Also, because it is reproducible,
+setting up a server with the same NixOS configuration
+will ensure that it runs the exact same software with the same configuration.
+Finally, if an update breaks your setup,
+NixOS makes rollbacks as easy as running one command or rebooting,
+so you can minimise downtime.
+
+## Requirements
+
+We will implement a small single-server deployment,
```

Comment:
small, single-server

Reply:
Given the low requirements given just a couple of lines later (1 CPU, 2 GB RAM) "small" could indeed describe "single-server" as "that server doesn't need to be beefy".

At the same time, considering "small" describing "deployment", I would understand it as "neglectible work, no need to actually talk about it, doesn't cause any problems", making this whole text obselete, or boring at least.

So personally I wouldn't consider a comma there.